### ISSUE TITLE
[Accessibility] Apply form fields contrast does not meed accessibility standards

### ISSUE THREAD

**Issue by alebiavati** [commitdev/commit.dev-proto#226]
### Description

Field background and placeholder text don't meet WCAG's minimum contrast standards. The contrast ratio should be at least 4.5:1.

### Context

- **Browser**:  any
- **OS**: any

### Steps to Reproduce

1. Visit https://commit-staging.dev/apply

### Possible Solution

@anikey can suggest new colors for background and placeholder text.

### Additional context / Screenshots

<img width="1046" alt="Screen Shot 2021-04-09 at 2 18 48 PM" src="https://user-images.githubusercontent.com/832191/114242141-41bfa480-993f-11eb-8e95-35ca3710c62a.png">
